detox/test/integration/utils/simplistic-runner.js

```diff
@@ -11,11 +12,16 @@ async function main() {
 }
 
 async function test1() {
+  log.info('Test 1');
```

Comment:
These log checks help to verify that the logger works as intended when Detox content is active.